extensions/positron-r/src/lsp.ts

```diff
@@ -152,28 +156,28 @@ export class ArkLsp implements vscode.Disposable {
 					break;
 				case State.Running:
 					if (this._initializing) {
-						LOGGER.info(`ARK (R ${this._version}) language client init successful`);
+						LOGGER.info(`${this.languageClientName} init successful`);
 						this._initializing = undefined;
-						if (this._client) {
+						if (this.client) {
 							// Register Positron-specific LSP extension methods
-							this.registerPositronLspExtensions(this._client);
+							this.registerPositronLspExtensions(this.client);
 						}
 						out.resolve();
 					}
 					this.setState(LspState.running);
 					break;
 				case State.Stopped:
 					if (this._initializing) {
-						LOGGER.info(`ARK (R ${this._version}) language client init failed`);
+						LOGGER.info(`${this.languageClientName} init failed`);
```

Comment:
FWIW these messages are synced up with the positron-python side to say very similar things with the same structure, i.e. on the Python side it says

```
traceInfo(`Python (${this._version}) language client init failed`);
```

Historically, the more in sync we can keep the implementations the easier it has been to reason about them